spec/models/guide_spec.rb

```diff
@@ -46,17 +46,23 @@
 
   describe "#latest_editable_edition" do
     it "returns the latest edition if it's not published" do
-      guide = Guide.create(slug: "/service-manual/ediatble", editions: [Generators.valid_edition])
+      guide = Guide.create(slug: "/service-manual/editable", editions: [Generators.valid_edition])
       expect(guide.latest_editable_edition).to eq guide.reload.latest_edition
     end
 
     it "returns an unsaved copy of the latest edition if the latter is published" do
-      guide = Guide.create(slug: "/service-manual/ediatble", editions: [Generators.valid_published_edition(title: "Agile Methodologies")])
+      guide = Guide.create(slug: "/service-manual/editable", editions: [Generators.valid_published_edition(title: "Agile Methodologies")])
```

Comment:
![i m 12 and what is this_fff216_4382158](https://cloud.githubusercontent.com/assets/247634/11334101/a2237e32-91ca-11e5-9076-4bb9b86f1bdc.gif)